x-pack/plugins/infra/public/pages/metrics/inventory_view/components/node_details/tabs/metrics/chart_section.tsx

```diff
@@ -0,0 +1,103 @@
+/*
```

Comment:
Nice refactor 👏 👏 

Since it's a new component, maybe we could add some unit tests for it? 😊

Reply:
Yeah, this component doesn't have, so not sure testing buys anything here. Anything in particular you would normally test in this scenario?

Reply:
That's a good question.

In this case, probably is enough with a snapshot test. Since snapshot testing is a bit controversial, I also worked in projects where I would check that each one of the components is rendered with the correct props. For example, I would test that there are as many `MetricExplorerSeriesChart` components as series passed as a prop or that there is an `Axis` component rendered with 'timestamp' as id, position in the bottom...

I'm aware that this can be seen as over-testing, but usually, tests like these are really quick to write and they can help to avoid future problems.

I'm happy to discuss more about this outside this PR, since it would make sense to be aligned as a team.